### DISCUSSION TITLE
Select Specific Type of Apk

### DISCUSSION THREAD

**Discussion by Exonymos** [ImranR98/Obtainium#1939] [Q&A]
Hello 👋, I want Obtainium to select apk manner in order like: armv8 > armv7 > normal. So if none of armv8 or armv7 are available only then the normal will be downloaded.
Example:
![Screenshot_20241101-201353.png](https://github.com/user-attachments/assets/08e2f13d-41aa-4f4d-a21a-39510803ed33)

**Comment by DwainZwerg** [comment]
This has not worked so far. You can create an issue.

**Comment by Julmik647** [comment]
It could be that you tell [Obtainium](https://github.com/ImranR98/Obtainium) to download relases with or without some text in them. For example: [Adguard](https://github.com/AdguardTeam/AdguardForAndroid/releases) adds TV for relases for Android TV, so you can download relases without "TV" on mobile and with "TV" on Android TV.
